test/gtest/ucs/test_async.cc

```diff
@@ -540,8 +540,7 @@ class local_event_add_handler : public local_event {
     local_event_add_handler(ucs_async_mode_t mode) :
         local_event(mode), m_event_set(false)
     {
-         int ret = pipe(m_pipefd);
-         ucs_assertv(0 == ret, "%m");
+         ucs_assertv(0 == pipe(m_pipefd), "%m");
```

Comment:
don't see how this fix is related..
also the call to pipe() will be removed in release mode - and we don't want that